src/themes/money/theme.json

```diff
@@ -1129,38 +1130,111 @@
   },
 
   "input": {
+    "base": {
+      "fontFamily": "base",
+      "fontSize": "base",
+      "lineHeight": 1.33,
+      "color": "text",
+      "appearance": "none",
+      "border": "none",
+      "background": "none",
+      "boxSizing": "border-box",
+      "display": "flex",
+      "flex": 1,
+      "position": "relative",
+      "verticalAlign": "middle",
+      "width": "100%",
+      "&:focus": {
+        "outline": "none"
+      },
+      "&:disabled": {
+        "outline": "none",
+        "border": "none",
+        "color": "primary",
+        "backgroundColor": "grey-05"
+      }
+    },
+    "wrapper": {
+      "backgroundColor": "white",
+      "boxSizing": "border-box",
+      "display": "flex",
+      "position": "relative",
+      "borderRadius": "4px"
+    },
+    "default": {
+      "color": "grey-40"
+    },
+    "error": {
+      "color": "error"
+    },
     "frozen": {
-      "backgroundColor": "grey-10",
-      "borderRadius": "3px"
+      "base": {
```

Comment:
this looks like a breaking change - we shouldn't be making breaking changes in themes